blazor/speeddial/accessibility.md

```diff
@@ -9,32 +9,69 @@ documentation: ug
 
 # Accessibility in Blazor SpeedDial Component
```

Comment:
Change it as small letter @SenthilarasuBalu 

Accessibility in Blazor SpeedDial **component**

Reply:
Correction added @IndrajithSrinivasan